BasicPlaybackSampleApp/BasicPlaybackSampleApp/AppDelegate.h

```diff
@@ -11,5 +11,7 @@
 
 @property (strong, nonatomic) UIWindow *window;
 
+@property (nonatomic,assign) int count ;
```

Comment:
please provide a comment explaining what this is for, and remove the space after count

for example "A counter for tracking the number of logs across all test case plays"